renpy/display/layout.py

```diff
@@ -477,31 +488,22 @@ def render(self, width, height, st, at):
         top_margin = compute_raw(self.style.top_margin, height)
         bottom_margin = compute_raw(self.style.bottom_margin, height)
 
```

Comment:
Worth keeping these.
```suggestion

        # For convenience and speed.
        cols = self.cols
        rows = self.rows

```